web/modules/custom/dpl_admin/src/Controller/AdminPages.php

```diff
@@ -0,0 +1,102 @@
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Url;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Custom admin pages.
+ */
+class AdminPages extends ControllerBase {
+
+  /**
+   * The entity type interface.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function __construct(EntityTypeManagerInterface $entity_type_manager) {
+    $this->entityTypeManager = $entity_type_manager;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container): AdminPages|static {
+    return new static(
+      $container->get('entity_type.manager'),
+    );
+  }
+
+  /**
+   * The 'add content' overview page, displaying options for the editors.
+   *
+   * @return array<mixed>
+   *   A render array, for displaying the page.
+   */
+  public function addContentPage(): array {
+
+    // 'Create from scratch' links - used in the admin block below.
+    $create_links = [
+      [
+        'title' => $this->t('Event series', [], ['context' => 'DPL admin UX']),
+        'url' => Url::fromRoute('entity.eventseries.add_page'),
+        'description' => $this->t('Use event series to create event instances - you can use series, both to create reoccurring events, and single events.', [], ['context' => 'DPL admin UX']),
+      ],
+    ];
+
+    $node_types = $this->entityTypeManager->getStorage('node_type')->loadMultiple();
+
+    // Adding a link to each node type.
+    foreach ($node_types as $node_type) {
+      $create_links[] = [
+        'title' => $node_type->label(),
+        'url' => Url::fromRoute('node.add', ['node_type' => $node_type->id()]),
+        'description' => $node_type->getDescription(),
+      ];
+    }
+
+    return [
+      '#theme' => 'container',
+      '#attributes' => [
+        'class' => ['layout-row', 'clearfix', 'dpl-add-content'],
+      ],
+      '#children' => [
+        [
+          '#theme' => 'admin_block',
+          '#prefix' => '<div class="layout-column layout-column--half">',
+          '#suffix' => '</div>',
+          '#block' => [
+            'title' => $this->t('Create content from scratch', [], ['context' => 'DPL admin UX']),
+            'content' => [
+              '#theme' => 'admin_block_content',
+              '#content' => $create_links,
+            ],
+          ],
+        ],
+        // Displaying the available clone templates.
+        [
+          '#theme' => 'admin_block',
+          '#prefix' => '<div class="layout-column layout-column--half">',
+          '#suffix' => '</div>',
+          '#block' => [
+            'title' => $this->t('Create content from templates', [], ['context' => 'DPL admin UX']),
+            'content' => [
+              '#type' => 'view',
+              '#name' => 'entity_clone_template',
+              '#display_id' => 'block',
+            ],
+          ],
+        ],
```

Comment:
Trying to clone the frontpage on the PR environment fails hard: https://varnish.pr-774.dpl-cms.dplplat01.dpl.reload.dk/entity_clone/node/4

```
Drupal\Component\Plugin\Exception\PluginNotFoundException: The "" entity type does not exist. in Drupal\Core\Entity\EntityTypeManager->getDefinition() (line 139 of core/lib/Drupal/Core/Entity/EntityTypeManager.php).

Drupal\Core\Entity\EntityTypeManager->getHandler() (Line: 192)
Drupal\Core\Entity\EntityTypeManager->getStorage() (Line: 41)
Drupal\entity_clone\EntityCloneClonableField->isClonable() (Line: 121)
Drupal\entity_clone\EntityClone\Content\ContentEntityCloneFormBase->formElement() (Line: 248)
Drupal\entity_clone\EntityClone\Content\ContentEntityCloneFormBase->getChildren() (Line: 222)
Drupal\entity_clone\EntityClone\Content\ContentEntityCloneFormBase->getRecursiveFormElement() (Line: 122)
Drupal\entity_clone\EntityClone\Content\ContentEntityCloneFormBase->formElement() (Line: 195)
Drupal\entity_clone\Form\EntityCloneForm->buildForm()
call_user_func_array() (Line: 536)
Drupal\Core\Form\FormBuilder->retrieveForm() (Line: 283)
Drupal\Core\Form\FormBuilder->buildForm() (Line: 73)
Drupal\Core\Controller\FormController->getContentResult()
call_user_func_array() (Line: 123)
Drupal\Core\EventSubscriber\EarlyRenderingControllerWrapperSubscriber->Drupal\Core\EventSubscriber\{closure}() (Line: 592)
Drupal\Core\Render\Renderer->executeInRenderContext() (Line: 124)
Drupal\Core\EventSubscriber\EarlyRenderingControllerWrapperSubscriber->wrapControllerExecutionInRenderContext() (Line: 97)
Drupal\Core\EventSubscriber\EarlyRenderingControllerWrapperSubscriber->Drupal\Core\EventSubscriber\{closure}() (Line: 181)
Symfony\Component\HttpKernel\HttpKernel->handleRaw() (Line: 76)
Symfony\Component\HttpKernel\HttpKernel->handle() (Line: 58)
Drupal\Core\StackMiddleware\Session->handle() (Line: 48)
Drupal\Core\StackMiddleware\KernelPreHandle->handle() (Line: 48)
Drupal\Core\StackMiddleware\ReverseProxyMiddleware->handle() (Line: 51)
Drupal\Core\StackMiddleware\NegotiationMiddleware->handle() (Line: 51)
Drupal\Core\StackMiddleware\StackedHttpKernel->handle() (Line: 704)
Drupal\Core\DrupalKernel->handle() (Line: 19)
```

Reply:
<img width="1392" alt="Monosnap Add new content | DPL CMS | Logged in 2024-02-19 16-35-53" src="https://github.com/danskernesdigitalebibliotek/dpl-cms/assets/73966/b427e549-7637-4c85-b7f5-9be7e9277074">

Reply:
i havent fixed the error, but i've done what you've suggested in the picture

Reply:
error is fixed now, it required a patch for dynamic_entity_reference